tests/auth.test.ts

```diff
@@ -97,6 +98,110 @@ describe('account registration', () => {
     verify(emailService.sendEmailVerification(user.email, user.firstName, anyString()))
       .never();
   });
+
+  test('User cannot register with a handle that is already in use', async () => {
+    const conn = await DatabaseConnection.get();
+    const existingMember = UserFactory.fake();
+
+    await new PortalState()
+      .createUsers(existingMember)
+      .write();
+
+    const user = {
+      email: 'acm@ucsd.edu',
+      firstName: 'ACM',
+      lastName: 'UCSD',
+      password: 'password',
+      major: UserFactory.major(),
+      graduationYear: UserFactory.graduationYear(),
+      handle: existingMember.handle,
+    };
+
+    const emailService = mock(EmailService);
+    when(emailService.sendEmailVerification(user.email, user.firstName, anyString()))
+      .thenResolve();
+    const authController = ControllerFactory.auth(conn, instance(emailService));
+    const registerRequest = { user };
+    await expect(authController.register(registerRequest, FactoryUtils.randomHexString()))
+      .rejects.toThrow('This handle is already in use.');
```

Comment:
nit: it's a bit unclear at a glance what the purpose of `FactoryUtils.randomHexString()` is here. Let's initialize a properly named variable for it instead and pass that variable into here.

Reply:
lowkey copied the code from the other auth test that's doing the same thing with the hex string so idk what it means

Reply:
Ok, let's just refactor that in a separate PR then. I think it's supposed to be the `accessCode` param.